src/sentry/static/sentry/app/utils/queryString.tsx

```diff
@@ -2,6 +2,8 @@ import queryString from 'query-string';
 import parseurl from 'parseurl';
 import isString from 'lodash/isString';
 
+import RESERVED_WORDS from './queryStringReservedWords';
```

Comment:
Should we use `TAGS` instead of `WORDS` so that it would be more specific to what is reserved.